src/polymer/iron-flex-layout-classes.ts

```diff
@@ -89,7 +89,18 @@ class IronFlexLayoutClasses extends HtmlRule {
       parsedDocument: ParsedHtmlDocument, document: Document,
       warnings: FixableWarning[]) {
     // Search in the dom-modules.
-    for (const domModule of document.getFeatures({kind: 'dom-module'})) {
+    for (const domModule of document.getFeatures({ kind: 'dom-module' })) {
+      const misplacedStyle = dom5.query(domModule.astNode, p.hasTagName('style'))
+      if (misplacedStyle) {
+        warnings.push(new FixableWarning({
```

Comment:
I think we should just silently continue here. They'll already get a warning for style-into-template.